auraescript/src/runtime.rs

```diff
@@ -0,0 +1,49 @@
+/* -------------------------------------------------------------------------- *\
```

Comment:
is this a generated file now?  it seems weird to have to repeat the contents of the proto.

Reply:
oh i guess we did that anyway.